source/ada/lsp-ada_driver.adb

```diff
@@ -47,9 +48,6 @@ with LSP.Stdio_Streams;
 
 procedure LSP.Ada_Driver is
 
-   function Getenv (Var : String) return String;
```

Comment:
Why do you move Getenv out of Ada_Driver?
I would prefer keep all environment reading before other threads started.

Reply:
Indeed, it's a leftover from a previous solution based on a environment variable to find the JSON databases. I'll put it back in ada-driver.adb